public/acuant/11.9.1/face_landmark_68_tiny_model-weights_manifest.json

```diff
@@ -0,0 +1,39 @@
+[
```

Comment:
(I'm commenting on this file because I can't comment on `application.yml.default` and I know you have a strong preference for comments on files.)

One note, I had to set `idv_acuant_sdk_version_default: '11.9.1'` in my local `application.yml` file when I was testing this, because `application.yml.default` still had `idv_acuant_sdk_version_default: '11.8.2'`.  When I didn't set this config variable, Acuant SDK didn't load.

I was able to successfully capture a selfie on droid after making this change! 🎉 👏🏻

Reply:
I will feel ready to approve this when
A) we have translations
B)`application.yml.default` includes `idv_acuant_sdk_version_default: '11.9.1'`. (I realize this is something devs would likely set already in their own local `application.yml` files, but I still think we should change it in `application.yml.default`.)

Reply:
A) Translations are in ([comment](https://github.com/18F/identity-idp/pull/9580/commits/170080afa059600490eb617c1609bc7680682537))
B) This should not happen in this PR, but I'll (keep) following up. It does effect this PR and is important in general.

Reply:
B) I made the related change [here](https://github.com/18F/identity-idp/pull/9651).